readme.md

```diff
@@ -21,7 +21,7 @@ Returns instance of HTTP/HTTPS agent, based on `options`.
 
 This what `agent` could return:
 
- * `undefined`, if http.Agent.defaultMaxSockets is `Infinity`
+ * `undefined`, if http.Agent.defaultMaxSockets is not `5`
```

Comment:
Done in https://github.com/floatdrop/infinity-agent/commit/eca2dd008607ed8c8800667cb8988912d9838fc7